.github/workflows/codeql.yml

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    name: Build
+    strategy:
```

Comment:
Pushed a change that does both

Reply:
Excuse me for any confusion. I mean "changing CodeQL's **job** name to `Analyze`" (located at L30 in this conversation's reviewed code block) instead of its **workflow** name (L20).

To be exactly, from `CodeQL/Build` to `CodeQL/Analyze`. The current version is `Analyze/Build`.

![image](https://github.com/apache/eventmesh/assets/41445332/1e84250c-9ea3-4074-be61-4fdab4bca196)

Reply:
Ah sorry, you said it clearly and I misunderstood. Will update this